docs/search-views/semantic-search/concepts/distance-type.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 20
+title: Distance Type
+---
+
+This page provides basic information about values in the **Distance Type** field, which is used in defining semantic search views in Macrometa Global Data Network (GDN).
```

Comment:
At the start can you say that the "distance" between two entities is the extent of similarity between them. More the distance, less is the similarity. Similarity search essentially returns the entities which have the least distance from the queried entity. (Feel free to reword)

Reply:
Great stuff, my only issue is that we don't use "similarity search" anywhere the user can see, so this sentence makes no sense: "Similarity search essentially returns the entities which have the least distance from the queried entity."

Could I replace "similarity" with "semantic" in that sentence and still have it be true?

Reply:
no we could/should say that the semantic search functionality in 17.17 is limited to similarity search, and then talk about the similarity search. I agree we don't show the term much in the UI although the way to exercise this functionality is using the c8ql function named "similar".

Reply:
Heh. Remember how in another PR you asked me about why I was calling this beta? It's because it doesn't seem like it's done yet! (Also, @romatastic  told me to.)

I'm thinking a note under the intro might be good. Are we planning to expand our semantic search beyond similarity search in the next release or two?

Reply:
yes, I believe that we will keep expanding it. Semantic search is more of an umbrella than just a feature.

Reply:
Updated the text. What do you think?